### DISCUSSION TITLE
Entity Tag

### DISCUSSION THREAD

**Discussion by philipprhrd** [Minestom/Minestom#1050] [Help] [answered]
I want to set the Entity Tag of an Entity with the Minestom Tag (e.g. Tag.Byte("OnGround"). I don't want to use the EntityMeta class!

**Comment by TheMode** [comment, marked as answer]
Tags on entities currently do not affect their meta at all. This is just a tool to more easily assign serializable data.

I have been thinking about making entities more data-driven and rely on tags, but this is not an immediate possibility.

**Reply by philipprhrd** [reply]
Would be helpful. Especially for further world deserialization with entities.
